### PR TITLE
Added Popcorn-Time support

### PR DESCRIPTION
**Pull request by JacopKane**
Made a new pull request from a clean fork.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,6 +275,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [Poedit](http://poedit.net/)
   - [PokerStars](http://www.pokerstars.com/)
   - [PopClip](http://pilotmoon.com/popclip/)
+  - [Popcorn-Time](https://popcorntime.io/)
```
Comment (lra) [inline README.md]: Looks like the official name is `Popcorn Time`

File: mackup/applications/popcorn-time.cfg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[application]
+name = Popcorn-Time
```
Comment (lra) [inline mackup/applications/popcorn-time.cfg]: let's use the official name

**Comment by lra** [comment]
can you add a line in the changelog?

File: mackup/applications/popcorn-time.cfg (hunk 2 above)
Comment (JacopKane) [inline mackup/applications/popcorn-time.cfg]: screenshot 2015-02-27 19 09 36
.app is with dash but I rather prefer yours as well.

**Comment by JacopKane** [comment]
@lra Sorry forgot that. Just added it as well.

**Pull request merged**